pipelines/validation/h1c_idr3_2/task_scripts/do_UNCALIBRATE.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/bash
+set -e
+
+# This script simulates and applies systematics to an input file.
+# Note that the path to where the perfectly calibrated files are stored
+# is contained in the mock_data.py script.
+
+# import common functions
+src_dir="$(dirname "$0")"
+source ${src_dir}/_common.sh
+
+# Parameters are set in the configuration file, here we define their positions,
+# which must be consistent with the config.
+# 1 - reference filename
+# 2 - sky component (e.g. foreground)
+# 3 - path to configuration files
+# 4 - path to where the uncalibration script lives
+# 5 - path to the IDR3.2 data products
+# 6 - path to where all the calibration products etc. will be saved
+
+fn="${1}"
+sky_cmp="${2}"
+config_path="${3}"
+script_dir="${4}"
+data_dir="${5}"
+save_dir_base="${6}"
+
+# Get the data filename correct, then append the data path.
+jd_int=$(get_int_jd `basename ${uvh5_fn}`)
```

Comment:
I'm not sure if you need to do `basename` here but it probably isn't hurting anything.

Reply:
I dunno, I think I just pulled it from one of the other scripts.

Reply:
Did a quick test, looks like the `basename` is necessary if `${uvh5_fn}` has anything preceding the file name. After looking through a few files, I see that I pulled this from the `do_REDCAL` script.